src/main/java/com/agilethought/atsceapi/validator/ValidatorUserService.java

```diff
@@ -0,0 +1,9 @@
+package com.agilethought.atsceapi.validator;
+
+import com.agilethought.atsceapi.model.User;
+
+public interface ValidatorUserService {
+	public void validateEmail(String email);
+	public void validatePassword(String password);
+	public void validateStatus(User user);
+}
```

Comment:
Same here, actually what was the intention behind creating a Service and then an Implementation class for it?